dcs/adapters/inbound/event_sub.py

```diff
@@ -81,8 +83,11 @@ async def _consume_files_to_register(self, *, payload: JsonObject) -> None:
             payload=payload, schema=event_schemas.FileInternallyRegistered
         )
 
+        object_id = str(uuid.uuid4())
```

Comment:
Choosing the object id is indeed part of the core logic and should not be done in the adapter. Maybe we can add another creation model as described below.